src/main/scala/org/broadinstitute/dsde/workbench/sam/service/TosService.scala

```diff
@@ -55,6 +55,16 @@ class TosService(val directoryDao: DirectoryDAO, val tosConfig: TermsOfServiceCo
       )
     )
 
+  def getTosText(docSet: Set[String]): IO[String] =
+    docSet match {
+      case set if set.isEmpty => IO.pure(termsOfServiceText)
+      case set if set.size == 1 && set.contains("privacyPolicy") => IO.pure(privacyPolicyText)
+      case set if set.size == 1 && set.contains("termsOfService") => IO.pure(termsOfServiceText)
+      case set if set.size == 2 && set.contains("privacyPolicy") && set.contains("termsOfService") =>
```

Comment:
This can be 
```scala
docSet match {
     case set if set.equals(Set("privacyPolicy")) => IO.pure(privacyPolicyText)
     case set if set.equals(Set("termsOfService")) => IO.pure(termsOfServiceText)
     case set if set.equals(Set("termsOfService", "privacyPolicy")) => IO.pure(termsOfServiceText + "\n\n" + privacyPolicyText)
     case _ => Set.empty
}
```